components/setting_picture.jsx

```diff
@@ -236,10 +252,12 @@ export default class SettingPicture extends Component {
         const img = this.renderImg();
 
         let confirmButtonClass = 'btn btn-sm';
+        let disableSaveButton = false;
```

Comment:
Would it read a bit better if the `disableSaveButton` variable was instead names `disableSaveButtonFocus`? Otherwise to me it looks like this will be used to completely disable the save button, even though further down it looks like it's only being used to set `tabIndex="-1"`.